src/components/Pools/RemoveFarm/index.tsx

```diff
@@ -113,10 +113,6 @@ const RemoveFarm = ({ stakingInfo, version, onClose, onLoading, onComplete, redi
         const hash = await withdrawCallback();
         setHash(hash);
 
-        if (onComplete) {
-          onComplete(100);
-        }
```

Comment:
this change breaks the current flow. 

The reason we had `onComplete(100);` here was because of the below flow.

When user remove 100% farm then once he complete it, we need to show him the flow to remove liquidity so user knows that after removing farm they need to also remove liquidity get their tokens back. 

After removing this line, flow is broken and user will not see remove liquidity UI after removing farm.